docs/V1/Partner/get_respondents.md

```diff
@@ -41,6 +42,7 @@ Avec les paramètres, ci-dessous.
 Clé              | valeur
 -----------------|------------------
 filters[status]  | accepted 
+filters[from]    | 1453649849 
 
 Le script retournera un code `200` et l'ensemble des candidats ayant été acceptés.
```

Comment:
If you change this exemple, don't forget to update that as well.
Something like:
"[...] ayant été acceptés et ajoutés après le ..."